resources/ext/cli/ca.erb

```diff
@@ -0,0 +1,30 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'fileutils'
+require 'puppetserver/ca/stub'
+
+command = ARGV[0]
+
+unless command == "import"
+  STDERR.puts "  'import' is the only valid subcommand, try:"
+  STDERR.puts "    `puppetserver ca import <key> <bundle> <crl>"
+  exit 1
+end
+
+private_key = ARGV[1]
+cert_bundle = ARGV[2]
+crl_chain   = ARGV[3]
+
+unless private_key && cert_bundle && crl_chain
+  STDERR.puts "The import subcommand requires three arguments: private-key, cert-bundle, crl, in that order."
+  exit 1
+end
+
+ca_dir = '/etc/puppetlabs/puppet/ssl/ca'
```

Comment:
I guess since we are hard-coding this, its not configurable. ^^

Reply:
It is, but the idea behind this PR was to create a "walking skeleton", "tracer bullet" esque, happy-path feature that proved we could integrated the puppetserver-cli gem and have an end to end demo of the idea.

Since this was originally written I've implemented most of the functionality for this ticket in https://github.com/puppetlabs/puppetserver-ca-cli/pull/1 and after that is merged this file will be re-written to look more like: https://github.com/puppetlabs/puppetserver-ca-cli/pull/1/files#diff-df26495b80ddb61db7362b0be03dd86e

Reply:
ok.  Sorry.  Missing context here